rest/utilities_testing_resttester.go

```diff
@@ -204,42 +208,48 @@ func (rt *RestTester) GetReplicationStatuses(queryString string) (statuses []db.
 //	  - user 'alice' created with star channel access
 //	  - http server wrapping the public API, remoteDBURLString targets the rt2 database as user alice (e.g. http://alice:pass@host/db)
 //	returned teardown function closes activeRT, passiveRT and the http server, should be invoked with defer
-func SetupSGRPeers(t *testing.T) (activeRT *RestTester, passiveRT *RestTester, remoteDBURLString string, teardown func()) {
+func SetupSGRPeers(t *testing.T, useMultipleCollections bool) (activeRT *RestTester, passiveRT *RestTester, remoteDBURLString string, teardown func()) {
```

Comment:
Is this acting more like a `useDefaultCollection` bool (if the condition on 222 is reversed) ?

Reply:
Yeah. I could change this if it bothers you, I'd just flip the booleans.